ajax/libs/mobx-react/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "mobx-react",
+  "filename": "index.min.js",
+  "version": "3.5.9",
+  "description": "React bindings for MobX. Create fully reactive components.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mobxjs/mobx-react.git"
+  },
+  "author": "Michel Weststrate",
+  "license": "MIT",
+  "homepage": "https://mobxjs.github.io/mobx",
+  "dependencies": {
+    "hoist-non-react-statics": "^1.2.0"
+  },
+  "keywords": [
+    "mobx",
+    "mobservable",
+    "react-component",
+    "react",
+    "reactjs",
+    "reactive"
+  ],
+  "npmName": "mobx-react",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "+(index|custom|native).+(js|map)"
```

Comment:
@kennynaoh 
Why do you need map here?
I saw there is no map file in the npm package.